docs/user/FAQ.rst

```diff
@@ -282,3 +282,46 @@ To inspect all output values, you can use either the
 
 For an example of how this works, have a look at this `notebook
 <https://nbviewer.jupyter.org/github/skorch-dev/skorch/blob/master/notebooks/Advanced_Usage.ipynb#Multiple-return-values-from-forward>`_.
+
+How can I perform gradient accumulation with skorch?
+----------------------------------------------------
+
+There is no direct option to turn on gradient accumulation (at least
+for now). However, with a few modifications, you can implement
+gradient accumulation yourself:
+
+
+.. code:: python
+
+    ACC_STEPS = 2  # number of steps to accumulate before updating weights
+
+    class GradAccNet(net_cls):
```

Comment:
Should we detail what `net_cls` is? Could this be `NeuralNetClassifer` to be slightly more concrete?